go/client/fuse_status_osx.go

```diff
@@ -0,0 +1,96 @@
+// +build darwin
+
+package client
+
+import (
+	"fmt"
+	"os"
+
+	keybase1 "github.com/keybase/client/go/protocol"
+	"github.com/keybase/go-kext"
+)
+
+func KeybaseFuseStatus(bundleVersion string) keybase1.FuseStatus {
+	status := keybase1.FuseStatus{
+		BundleVersion: bundleVersion,
+		InstallStatus: keybase1.InstallStatus_UNKNOWN,
+		InstallAction: keybase1.InstallAction_UNKNOWN,
+	}
+
+	var kextInfo *kext.Info
+
+	// Check osxfuse 3.x
+	path3 := "/Library/Filesystems/osxfuse.fs"
```

Comment:
I started re-factoring this and it became harder to read/process... It's filling in stuff and then may error and return at any moment. I think it's probably ok to be repetitive here cause it's super obvious whats happening?